`make serve` yields "Error: Cannot find modules 'clean', 'start:webpack', or 'start:docs'"

Following the README.md:

```shell
git clone https://github.com/plone/mockup.git
cd mockup
make install
make serve
```

Results in this error message:

```console
Makefile:25: warning: overriding commands for target `stamp-yarn'
node_modules/@patternslib/dev/Makefile:23: warning: ignoring old commands for target `stamp-yarn'
Makefile:25: warning: overriding commands for target `install'
node_modules/@patternslib/dev/Makefile:23: warning: ignoring old commands for target `install'
Makefile:45: warning: overriding commands for target `bundle'
node_modules/@patternslib/dev/Makefile:63: warning: ignoring old commands for target `bundle'
Makefile:50: warning: overriding commands for target `bundle-pre'
node_modules/@patternslib/dev/Makefile:50: warning: ignoring old commands for target `bundle-pre'
npx yarn run start
yarn run v1.22.21
$ npm-run-all clean --parallel start:*
Watching /Users/stevepiercy/projects/Plone/mockup and all sub-directories not excluded by your .gitignore. Will not monitor dotfiles.
Found & ignored ./.git ; is listed in .gitignore
Found & ignored ./.github ; is listed in .gitignore
Found & ignored ./.husky ; is listed in .gitignore
Found & ignored ./_data ; is listed in .gitignore
Found & ignored ./_includes ; is listed in .gitignore
Found & ignored ./docs ; is listed in .gitignore
Found & ignored ./node_modules ; is listed in .gitignore
Found & ignored ./patches ; is listed in .gitignore
Found & ignored ./src ; is listed in .gitignore
Found & ignored ./CHANGES.md ; is listed in .gitignore
Found & ignored ./LICENSE.md ; is listed in .gitignore
Found & ignored ./Makefile ; is listed in .gitignore
Found & ignored ./README.md ; is listed in .gitignore
Found & ignored ./babel.config.js ; is listed in .gitignore
Found & ignored ./commitlint.config.js ; is listed in .gitignore
Found & ignored ./index.html ; is listed in .gitignore
Found & ignored ./jest.config.js ; is listed in .gitignore
Found & ignored ./package.json ; is listed in .gitignore
Found & ignored ./prettier.config.js ; is listed in .gitignore
Found & ignored ./stamp-yarn ; is listed in .gitignore
Found & ignored ./webpack.config.js ; is listed in .gitignore
Found & ignored ./widgets.pot ; is listed in .gitignore
Found & ignored ./yarn.lock ; is listed in .gitignore

Starting: clean
node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module '/Users/stevepiercy/projects/Plone/mockup/clean'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.10.0
Watching /Users/stevepiercy/projects/Plone/mockup and all sub-directories not excluded by your .gitignore. Will not monitor dotfiles.
Found & ignored ./.git ; is listed in .gitignore
Found & ignored ./.github ; is listed in .gitignore
Found & ignored ./.husky ; is listed in .gitignore
Found & ignored ./_data ; is listed in .gitignore
Found & ignored ./_includes ; is listed in .gitignore
Watching /Users/stevepiercy/projects/Plone/mockup and all sub-directories not excluded by your .gitignore. Will not monitor dotfiles.
Found & ignored ./docs ; is listed in .gitignore
Found & ignored ./node_modules ; is listed in .gitignore
Found & ignored ./patches ; is listed in .gitignore
Found & ignored ./src ; is listed in .gitignore
Found & ignored ./CHANGES.md ; is listed in .gitignore
Found & ignored ./LICENSE.md ; is listed in .gitignore
Found & ignored ./Makefile ; is listed in .gitignore
Found & ignored ./README.md ; is listed in .gitignore
Found & ignored ./babel.config.js ; is listed in .gitignore
Found & ignored ./commitlint.config.js ; is listed in .gitignore
Found & ignored ./index.html ; is listed in .gitignore
Found & ignored ./jest.config.js ; is listed in .gitignore
Found & ignored ./package.json ; is listed in .gitignore
Found & ignored ./prettier.config.js ; is listed in .gitignore
Found & ignored ./stamp-yarn ; is listed in .gitignore
Found & ignored ./webpack.config.js ; is listed in .gitignore
Found & ignored ./widgets.pot ; is listed in .gitignore
Found & ignored ./.git ; is listed in .gitignore
Found & ignored ./yarn.lock ; is listed in .gitignore
Found & ignored ./.github ; is listed in .gitignore
Found & ignored ./.husky ; is listed in .gitignore
Found & ignored ./_data ; is listed in .gitignore
Found & ignored ./_includes ; is listed in .gitignore
Found & ignored ./docs ; is listed in .gitignore
Found & ignored ./node_modules ; is listed in .gitignore
Found & ignored ./patches ; is listed in .gitignore
Found & ignored ./src ; is listed in .gitignore
Found & ignored ./CHANGES.md ; is listed in .gitignore
Found & ignored ./LICENSE.md ; is listed in .gitignore
Found & ignored ./Makefile ; is listed in .gitignore
Found & ignored ./README.md ; is listed in .gitignore
Found & ignored ./babel.config.js ; is listed in .gitignore
Found & ignored ./commitlint.config.js ; is listed in .gitignore
Found & ignored ./index.html ; is listed in .gitignore
Found & ignored ./jest.config.js ; is listed in .gitignore
Found & ignored ./package.json ; is listed in .gitignore
Found & ignored ./prettier.config.js ; is listed in .gitignore
Found & ignored ./stamp-yarn ; is listed in .gitignore
Found & ignored ./webpack.config.js ; is listed in .gitignore
Found & ignored ./widgets.pot ; is listed in .gitignore
Found & ignored ./yarn.lock ; is listed in .gitignore

Starting: start:webpack

Starting: start:docs
node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module '/Users/stevepiercy/projects/Plone/mockup/start:webpack'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.10.0
node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module '/Users/stevepiercy/projects/Plone/mockup/start:docs'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.10.0
✨  Done in 2.44s.
```

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
         "@testing-library/jest-dom": "^6.4.2",
         "@types/sinon": "^10.0.20",
         "css.escape": "^1.5.1",
-        "npm-run-all": "^4.1.5",
+        "npm-run-all2": "^7.0.1",
         "rimraf": "^4.1.3",
         "sinon": "^15.2.0",
         "svelte": "^4.2.19",
@@ -102,7 +102,7 @@
         "collect:externaldocs": "copyup -V node_modules/@patternslib/patternslib/src/pat/**/documentation.md node_modules/@patternslib/pat-*/README.md node_modules/pat-*/README.md docs/external/",
         "start:webpack": "NODE_ENV=development webpack serve --config webpack.config.js",
         "start:docs": "ELEVENTY_ENV=development npx @11ty/eleventy --serve --port 8001 ",
-        "start": "npm-run-all clean --parallel start:*",
+        "start": "npm run clean && npm-run-all --parallel start:*",
         "test": "jest",
         "testwatch": "jest --watch",
         "i18n": "node src/i18n.js"

